src/scss/base/_reset.scss

```diff
@@ -16,11 +17,13 @@ img {
 }
 
 * {
-  -webkit-box-sizing: border-box;
-  -moz-box-sizing: border-box;
   box-sizing: border-box;
 }
 
+b, strong {

```

Comment:
This is for wysywig editors when they will add bold and it will blured fonts when it will by automatically font-weight: bold; by browser.